spark-plugins/src/main/java/co/cask/hydrator/plugin/spark/TwitterStreamingSource.java

```diff
@@ -31,8 +31,6 @@
 import org.apache.spark.streaming.api.java.JavaDStream;
 import org.apache.spark.streaming.api.java.JavaStreamingContext;
 import org.apache.spark.streaming.twitter.TwitterUtils;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
```

Comment:
These were unused imports that IntelliJ removed automatically while refactoring.